drake/systems/Simulation.h

```diff
@@ -112,7 +113,7 @@ void simulate(const System& sys, double ti, double tf,
     try {
       handle_realtime_factor(start, t, options.realtime_factor,
                              options.timeout_seconds);
-    } catch(std::runtime_error err) {
+    } catch (std::runtime_error err) {
```

Comment:
Catch by-reference.
http://stackoverflow.com/questions/2023032/catch-exception-by-pointer-in-c#answer-2023045